src/main/java/org/zalando/nakadi/config/SecurityConfiguration.java

```diff
@@ -60,6 +60,7 @@ public void configure(final HttpSecurity http) throws Exception {
                     .antMatchers(POST, "/event-types/**").access(hasScope(eventTypeWriteScope))
                     .antMatchers(PUT, "/event-types/**").access(hasScope(eventTypeWriteScope))
                     .antMatchers(POST, "/subscriptions/**").access(hasScope(eventStreamReadScope))
+                    .antMatchers(GET, "/subscriptions/**").access(hasScope(eventStreamReadScope))
```

Comment:
Up there you have
.antMatchers(GET, "/subscriptions/*/events/**").access(hasScope(eventStreamReadScope))

I am not sure but I think this line covers this case
.antMatchers(GET, "/subscriptions/**").access(hasScope(eventStreamReadScope))

What do you think ?